content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -36,8 +37,12 @@ In the RBAC API, a role contains rules that represent a set of permissions.
 Permissions are purely additive (there are no "deny" rules).
 A role can be defined within a namespace with a `Role`, or cluster-wide with a `ClusterRole`.
 
-A `Role` can only be used to grant access to resources within a single namespace.
-Here's an example `Role` in the "default" namespace that can be used to grant read access to pods:
+A Role sets permissions within a {{< glossary_tooltip text="namespace" term_id="namespace" >}},
+whereas a ClusterRole applies across all namespaces in your
```

Comment:
The previous text was more accurate. ClusterRoles can 1) define permissions on namespaced resources and be granted within individual namespaces, 2) define permissions on namespaced resources and be granted across all namespaces, 3) define permissions on cluster-scoped resources. Separating the discussion of role and clusterrole objects is probably clearer.